MODULE.bazel

```diff
@@ -0,0 +1,6 @@
+###############################################################################
+# Bazel now uses Bzlmod by default to manage external dependencies.
```

Comment:
This is auto generated by bazel 7 so I think we need to leave it otherwise it will keep getting re-generated. We could add it to `.gitignore` if you'd prefer that?

Reply:
Nah, leave it then. It felt autogenerated but wanted to make sure.